2001-3000/LIT2834RepCh118.xml

```diff
@@ -4,7 +4,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title xml:id="t1">Malkǝʾ-hymn to king Naʾakʷǝto La-ʾab</title>
+            <title xml:id="t1" xml:lang="gez">መልክአ፡ ነአኰቶ፡ ለአብ፡</title>
+            <title corresp="#t1" xml:lang="gez" type="normalized">Malkǝʾa Naʾakkʷǝto La-ʾab</title>
+            <title corresp="#t1" xml:lang="en" type="main">Image of Naʾakkʷǝto La-ʾab</title>
```

Comment:
If Naʾakʷǝto La-ʾab is an incorrect form, I do not see any necessity in it.

Reply:
ok, updated